.circleci/config.yml

```diff
@@ -48,6 +48,9 @@ jobs:
             - node_modules
             - ~/.cache/Cypress
       - run: npm run build
+      - run:
+          name: check php code style
+          command: vendor/bin/php-cs-fixer fix --dry-run --no-interaction --verbose
```

Comment:
Not super used to configuring CircleCI so lemme know if this looks ok. Essentially doing a `--dry-run` since if it tried to fix the code, it'd need to also commit the changes, etc. This at least will error out and signal that there's a style error.

Reply:
Looks good to me! 👍